pkg/ddevapp/utils.go

```diff
@@ -490,6 +490,33 @@ func GetProjectNamesFunc(status string, numArgs int) func(*cobra.Command, []stri
 	}
 }
 
+// GetServiceNamesFunc returns a function for autocompleting service names for service flag.
+// If existingOnly is true, only names of existing services will be returned.
+func GetServiceNamesFunc(existingOnly bool) func(*cobra.Command, []string, string) ([]string, cobra.ShellCompDirective) {
+	return func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+		app, err := GetActiveApp("")
```

Comment:
This makes it not work right everywhere because it's getting the current app, rather than a possibly named app. For example, `ddev ssh craftdemo -s <tab>` gets the services for the current app instead of "creaftdemo".

Reply:
Thanks, I rarely use this feature with project arguments, and I forgot about it.